### PR TITLE
chore) expose x-ms-client-id property in MapsAccount

### PR DESCRIPTION
**Pull request by SudhindraKovalam**
## Description

Link to the rest-api-specs repo PR: https://github.com/Azure/azure-rest-api-specs/pull/4946

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR  
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
Please fix failing tests

**Comment by SudhindraKovalam** [comment]
@dsgouda , the tests have been fixed. Can you please take a look ?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, please address the comment

File: src/SDKs/Maps/Management.Maps/Microsoft.Azure.Management.Maps.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <PackageId>Microsoft.Azure.Management.Maps</PackageId>
     <PackageReleaseNotes>This package replaces Microsoft.Azure.Management.LocationBasedServices. Azure Location Based Services is now Azure Maps.</PackageReleaseNotes>
     <Description>Microsoft Azure Management Maps Library</Description>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
     <AssemblyName>Microsoft.Azure.Management.Maps</AssemblyName>
     <PackageTags>Microsoft Azure Maps management;Location Services;Location Services management;REST HTTP client;windowsazureofficial;netcore451511</PackageTags>
     <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```
[the remark this answers is not in the document]
Comment (SudhindraKovalam) [reply src/SDKs/Maps/Management.Maps/Microsoft.Azure.Management.Maps.csproj]: Updated the release notes

**Review by dsgouda — APPROVED** [review]
LGTM will merge once CIs pass

**Pull request merged**